assets/lokomotive-kubernetes/bootkube/resources/charts/webhook-admission/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
As an internal chart, do we need all those helpers?

Reply:
Can remove it, will do